VERSION.cmake

```diff
@@ -1,11 +1,11 @@
 SET( VERSION_MAJOR "2" )
-SET( VERSION_MINOR "43" )
-SET( VERSION_PATCH "6" )
+SET( VERSION_MINOR "44" )
+SET( VERSION_PATCH "0" )
 SET( VERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}" )
 
 ##### This is need for the libyui core, ONLY.
 ##### These will be overridden from exports in LibyuiConfig.cmake
-SET( SONAME_MAJOR "5" )
+SET( SONAME_MAJOR "0" )
```

Comment:
Shoudn't we set it to 10? I'll provide PR for `libyui-qt-graph` in a minute.

Reply:
As written in the comment above these values are overridden by LibyuiConfig.cmake. To avoid keeping it in sync, which might be easily overlooked as you can see, I'd rather use some meaningless number which we can just leave without updating it every time.